src/dos/cdrom_ioctl_linux.cpp

```diff
@@ -113,4 +320,159 @@ bool CDROM_Interface_Ioctl::ReadSectorsHost([[maybe_unused]] void *buffer,
 	return false; /*TODO*/
 }
 
+void CDROM_Interface_Ioctl::InitAudio(int device_number)
+{
+	if (mixer_channel) {
+		return;
+	}
+
+	std::string name = std::string(MixerChannelNamePrefix) +
+	                   std::to_string(device_number);
+	audio_buffer.resize(AudioBufferBytes / 2);
```

Comment:
I don't quite like this; can you derive the number of _samples_ instead from the constants and call this `AudioBufferSamples`? Then you don't need to divide by two.

To be clear:

1 sample frame = 2 samples (always stereo)
1 sample = int16_t